mantidproject/mantid
# Addition of a ExtractMonitors

Adds a separate monitor spectra algorithm, which takes a workspace with detectors and monitors as input, and outputs a workspace with just detectors, and/or a workspace with just monitors.

**To test:**
- Load any file which creates detectors and monitors in a single workspace (e.g. LOQ49886.nxs files in usage data), and run the algorithm with different options set to test behaviour
- Load any file which creates detectors, and monitors in a linked workspace (e.g. CNCS_7860_event.nxs with LoadMonitors selected) and check an error is thrown, mentioning the name of the monitor workspace
- Can also test using CreateSampleWorkspace, with different numbers of monitors, detectors etc.
- Check documentation is clear

Fixes #17138.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
